src/backend/joanie/tests/core/test_api_admin_order_group.py

```diff
@@ -0,0 +1,147 @@
+"""
+Test suite for OrderGroup Admin API.
+"""
+
+from operator import itemgetter
+
+from django.test import TestCase
+
+from joanie.core import factories, models
+
+
+class OrderGroupAdminApiTest(TestCase):
+    """
+    Test suite for OrderGroup Admin API.
```

Comment:
@samy-aitouakli I think we should stick to what is done elsewhere:
- test in the right order: read-list, read-detail, create, update, patch, delete
- anonymous/authenticated

Reply:
I reopen this discussion as this has not been fixed.

Reply:
Oh, I didn't notice the mention of "order" the first time, I thought that was because some verbs were missing